src/neighbors.cc

```diff
@@ -1431,13 +1435,15 @@ peerCountMcastPeersStart(void *data)
              psstate,
              Config.Timeout.mcast_icp_query / 1000.0, 1);
     p->mcast.flags.counting = true;
+    CodeContext::Reset();
```

Comment:
I think we should reset before peerCountMcastPeersSchedule() because the scheduling (and its context) does not relate to this ICP transaction. In other words, we should not propagate the context of the current transaction to a future transaction (despite the fact that currently scheduling an event does not preserve the current transaction context).